package.json

```diff
@@ -19,5 +18,8 @@
   ],
   "author": "Razorpay Developers <developers@razorpay.com> (https://razorpay.com/)",
   "license": "MIT",
-  "homepage" : "https://github.com/razorpay/react-native-razorpay#readme"
+  "homepage": "https://github.com/razorpay/react-native-razorpay#readme",
+  "dependencies": {
+    "react-native": "^0.39.1"
+  }
```

Comment:
Can we keep this as optional? My guess is that since this a breaking change (kind of), not many people will upgrade to this version of react-native. We should not be forcing this.